interfaces/src/db.ts

```diff
@@ -105,11 +105,10 @@ export interface Query {
 
 // tslint:disable-next-line:no-empty-interface
 export interface ClientContext {
+  appId: string;
```

Comment:
Occurs to me: This might be better inside another object: it's the ID for what you want to access.  E.g. that subobject may expand to include collection ID.

Reply:
We could just add `collection` to the top level when we support it.